tapir/log/views.py

```diff
@@ -50,3 +59,80 @@ def create_text_log_entry(request, **kwargs):
 
     form.save()
     return safe_redirect(request.GET.get("next"), "/", request)
+
+
+class LogTable(django_tables2.Table):
+    entry = django_tables2.Column(
+        empty_values=(), accessor="as_leaf_class__render", verbose_name="Message"
+    )
+
+    class Meta:
+        model = LogEntry
+        template_name = "django_tables2/bootstrap.html"
+        fields = ["created_date", "actor", "user", "entry"]
+
+    def render_actor(self, value):
+        return "%s" % value.get_display_name()
+
+
+class LogFilter(django_filters.FilterSet):
+    def __init__(self, user=None, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        if not self.request.user.has_perm("coop.view"):
+            userfield_choices = [
+                (
+                    TapirUser.objects.get(username=self.request.user).id,
+                    TapirUser.objects.get(
+                        username=self.request.user
+                    ).get_display_name(),
+                )
+            ]
+        else:
+            userfield_choices = [
+                (
+                    x,
+                    TapirUser.objects.get(pk=x).get_display_name(),
+                )
+                for x in TapirUser.objects.all()
+                .values_list("id", flat=True)
+                .distinct()
+                .order_by("username")
+                if x is not None
+            ]
```

Comment:
I think you can simplify this, see the example below.
Even better would be to use the fancy fields where you can search by name, you can see examples in PR #252 

```suggestion
           userfield_choices = [
                (
                    user.id,
                    user.get_display_name(),
                )
                for user in TapirUser.objects.all().order_by("username")
            ]
```